hazelcast-jet-core/src/test/java/com/hazelcast/jet/pipeline/test/TestSourcesTest.java

```diff
@@ -50,29 +50,21 @@ public void test_items() {
     }
 
     @Test
-    public void test_long_batch_range() {
-        int upperValueRange = 10_000;
-        Long[] input = LongStream.range(0, upperValueRange).boxed().toArray(Long[]::new);
-        List<Long> expected = Arrays.asList(input);
-
-        TestSources.batchStageForLongRange(p, upperValueRange, 10)
-                .apply(
-                        Assertions.assertAnyOrder(expected)
-                );
-
-        jet().newJob(p).join();
-    }
-
-    @Test
-    public void test_long_stream_range() throws Throwable {
-        int upperValueRange = 10_000;
-        Long[] input = LongStream.range(0, upperValueRange).boxed().toArray(Long[]::new);
+    public void test_stream_source_long() throws Throwable {
```

Comment:
Test method names shouldn't use `_` arbitrarily. We use the `camelCase` except for the special format: `when_preCondition_then_postCondition`. The latter naming scheme is preferable for test methods.

Reply:
Done w/ 8fcf10859c9274eca0f80fe5f7b45bac375000dc